contrib/builder/rpm/generate.sh

```diff
@@ -57,7 +61,11 @@ for version in "${versions[@]}"; do
 		sqlite-devel # for "sqlite3.h"
 		tar # older versions of dev-tools don't have tar
 	)
-	echo "RUN yum install -y ${packages[*]}" >> "$version/Dockerfile"
+	if [ "$from" == "oraclelinux:7" ]; then
+		echo "RUN yum install --enablerepo=ol7_optional_latest -y ${packages[*]}" >> "$version/Dockerfile"
```

Comment:
I can check internally if we're OK with enabling the repo in our Docker image, but we're following the convention of our upstream vendor with the split of the main/optional repos and having the optional repo disabled/unsubscribed by default. 

I see this as the equivalent to the CentOS-specific stuff to switch out systemd-container (which is more of a hack, considering systemd-container isn't provided by the upstream vendor).